src/test/java/org/apache/sling/jcr/repoinit/PrincipalBasedAclTest.java

```diff
@@ -461,10 +461,16 @@ public void  principalAclNotAvailable() throws Exception {
         try {
             // create service user outside of supported tree for principal-based access control
             U.parseAndExecute("create service user otherSystemPrincipal");
+            // principal-based ac-setup must fail as service user is not located below supported path
             String setup = "set principal ACL for otherSystemPrincipal \n"
                             + "allow jcr:read on " + path + "\n"
                             + "end";
-            U.parseAndExecute(setup);
+            try {
+                U.parseAndExecute(setup);
+                fail("Setting a principal ACL outside a supported path must not succeed");
+            } catch (RuntimeException e) {
+                // expected
```

Comment:
i would verify the nature of the exception making sure that it is not some other runtime exception but really the expected one.

Reply:
done in https://github.com/apache/sling-org-apache-sling-jcr-repoinit/pull/14/commits/cfadffb925ae3da29838a5f7a6a15630a147fbec